### PR TITLE
Update to latest available resources

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

File: provider/cmd/pulumi-gen-aws-native/supported-types.txt [hunk 1]
```diff
@@ -133,7 +135,6 @@ AWS::EFS::AccessPoint
 AWS::EFS::FileSystem
 AWS::EFS::MountTarget
 AWS::EKS::Addon
-AWS::EKS::Cluster
```
Comment (viveklak) [inline provider/cmd/pulumi-gen-aws-native/supported-types.txt:136]: Are we cool with this going away?

Comment (lblackstone) [reply provider/cmd/pulumi-gen-aws-native/supported-types.txt:136]: They apparently rolled this back due to some bug, so I don't think it's working now anyway.

Comment (viveklak) [reply provider/cmd/pulumi-gen-aws-native/supported-types.txt:136]: Oh. So is https://github.com/pulumi/examples/tree/master/aws-native-ts-ecs broken?

Comment (lblackstone) [reply provider/cmd/pulumi-gen-aws-native/supported-types.txt:136]: No, this is EKS, not ECS.

Comment (viveklak) [reply provider/cmd/pulumi-gen-aws-native/supported-types.txt:136]: Yes - I realized that just now 🤦🏾.

**Review by viveklak — APPROVED** [review]

**Pull request merged**